physics/electric_power.py

```diff
@@ -0,0 +1,60 @@
+"""
+Calculating Electric Power
+Description : Electric power can be said to be the rate at which
```

Comment:
The provided code calculates the electric power for DC circuits based on the given input parameters, which include the potential difference (voltage), current, and resistance. 

You should change PR name as DC Electric Power formula.

Reply:
Thankyou for the feedback